### ISSUE TITLE
Provide clickable interactions to run a test (file, func, package)

### ISSUE THREAD

**Issue by piotrkubisa** [yields/sublime-go#22]
In the VSCode you can press `run test` or  `debug test` that is displayed on the top of the test function.  I believe it is possible to add `run test` in Phantom as in https://github.com/aziz/SublimeHyperClick

**Comment by yields** [comment]
Yeah that'd be cool as a toggle/option, I'm on sublime's discord and there's a lot of nice API additions for nicer clicking, control etc..

Personally, I think this pkg shouldn't depend on other pkgs, it makes things 10x harder to test and it's already hard as it is ;).

**Comment by piotrkubisa** [comment]
Indeed. I think it would be nice to have it as a separated package, because it will be too much clutter. It is easy to install an _extra_ package that will do specialized to just running and testing while other related to editing.

**Comment by yields** [comment]
Good news is that sublime-text v4 will now allow minihtml links to invoke commands, so presumably it will look like:

```html
<a href="subl:go_run_test">Run</a>
```

**Comment by piotrkubisa** [comment]
Awesome! That sounds really easy to make now. Perfect time to look for some free time to implement this ;)
